!!python/object:huggingface_hub.community.DiscussionWithDetails
author: Khaledelsaka
conflicting_files: null
created_at: 2023-03-28 02:39:33+00:00
diff: null
endpoint: https://huggingface.co
events:
- !!python/object:huggingface_hub.community.DiscussionComment
  _event:
    author:
      avatarUrl: /avatars/92d1f678a63040d607da31c0855a9790.svg
      fullname: Khaled El-Saka
      isHf: false
      isOrgMember: false
      isOwner: false
      isPro: false
      name: Khaledelsaka
      type: user
    createdAt: '2023-03-28T03:39:33.000Z'
    data:
      edited: false
      editors:
      - Khaledelsaka
      hidden: false
      isReport: false
      latest:
        author:
          avatarUrl: /avatars/92d1f678a63040d607da31c0855a9790.svg
          fullname: Khaled El-Saka
          isHf: false
          isPro: false
          name: Khaledelsaka
          type: user
        html: '<p>Thanks for sharing ^^<br>I am getting this error "The model_type
          ''pix2struct'' is not recognized. It could be a bleeding edge model, or
          incorrect" when I try any of the checkpoints for inference.<br><a rel="nofollow"
          href="https://cdn-uploads.huggingface.co/production/uploads/640a0ed29989bcb11723ae95/waempOGZbu2EUeMSIs6WD.png"><img
          alt="Screenshot 2023-03-28 053518.png" src="https://cdn-uploads.huggingface.co/production/uploads/640a0ed29989bcb11723ae95/waempOGZbu2EUeMSIs6WD.png"></a></p>

          '
        raw: "Thanks for sharing ^^\r\nI am getting this error \"The model_type 'pix2struct'\
          \ is not recognized. It could be a bleeding edge model, or incorrect\" when\
          \ I try any of the checkpoints for inference.\r\n![Screenshot 2023-03-28\
          \ 053518.png](https://cdn-uploads.huggingface.co/production/uploads/640a0ed29989bcb11723ae95/waempOGZbu2EUeMSIs6WD.png)\r\
          \n"
        updatedAt: '2023-03-28T03:39:33.791Z'
      numEdits: 0
      reactions:
      - count: 2
        reaction: "\U0001F44D"
        users:
        - abrichr
        - Ramlov
    id: 642261752cc2b3c39e861dfd
    type: comment
  author: Khaledelsaka
  content: "Thanks for sharing ^^\r\nI am getting this error \"The model_type 'pix2struct'\
    \ is not recognized. It could be a bleeding edge model, or incorrect\" when I\
    \ try any of the checkpoints for inference.\r\n![Screenshot 2023-03-28 053518.png](https://cdn-uploads.huggingface.co/production/uploads/640a0ed29989bcb11723ae95/waempOGZbu2EUeMSIs6WD.png)\r\
    \n"
  created_at: 2023-03-28 02:39:33+00:00
  edited: false
  hidden: false
  id: 642261752cc2b3c39e861dfd
  type: comment
- !!python/object:huggingface_hub.community.DiscussionComment
  _event:
    author:
      avatarUrl: /avatars/0de4e8ff69eb80ef849c791f32a604f4.svg
      fullname: vinaykumar
      isHf: false
      isOrgMember: false
      isOwner: false
      isPro: false
      name: vinay-k12
      type: user
    createdAt: '2023-03-30T19:28:40.000Z'
    data:
      edited: false
      editors:
      - vinay-k12
      hidden: false
      isReport: false
      latest:
        author:
          avatarUrl: /avatars/0de4e8ff69eb80ef849c791f32a604f4.svg
          fullname: vinaykumar
          isHf: false
          isPro: false
          name: vinay-k12
          type: user
        html: '<p>Same issue for me too</p>

          '
        raw: Same issue for me too
        updatedAt: '2023-03-30T19:28:40.950Z'
      numEdits: 0
      reactions: []
    id: 6425e2e8a545bd6b38e154d9
    type: comment
  author: vinay-k12
  content: Same issue for me too
  created_at: 2023-03-30 18:28:40+00:00
  edited: false
  hidden: false
  id: 6425e2e8a545bd6b38e154d9
  type: comment
- !!python/object:huggingface_hub.community.DiscussionComment
  _event:
    author:
      avatarUrl: /avatars/ce90534887f167609daf8917d6ec4f9e.svg
      fullname: Chintan Gotecha
      isHf: false
      isOrgMember: false
      isOwner: false
      isPro: false
      name: gaussfer
      type: user
    createdAt: '2023-05-09T05:02:33.000Z'
    data:
      edited: false
      editors:
      - gaussfer
      hidden: false
      isReport: false
      latest:
        author:
          avatarUrl: /avatars/ce90534887f167609daf8917d6ec4f9e.svg
          fullname: Chintan Gotecha
          isHf: false
          isPro: false
          name: gaussfer
          type: user
        html: '<p>+1 same issue.</p>

          '
        raw: +1 same issue.
        updatedAt: '2023-05-09T05:02:33.505Z'
      numEdits: 0
      reactions: []
    id: 6459d3e939e6aea69ccc6809
    type: comment
  author: gaussfer
  content: +1 same issue.
  created_at: 2023-05-09 04:02:33+00:00
  edited: false
  hidden: false
  id: 6459d3e939e6aea69ccc6809
  type: comment
- !!python/object:huggingface_hub.community.DiscussionComment
  _event:
    author:
      avatarUrl: /avatars/cf57e3c6d0a9fa281c4a982b0c93558a.svg
      fullname: Kamalraj M M
      isHf: false
      isOrgMember: false
      isOwner: false
      isPro: false
      name: Kamaljp
      type: user
    createdAt: '2023-05-09T08:15:07.000Z'
    data:
      edited: false
      editors:
      - Kamaljp
      hidden: false
      isReport: false
      latest:
        author:
          avatarUrl: /avatars/cf57e3c6d0a9fa281c4a982b0c93558a.svg
          fullname: Kamalraj M M
          isHf: false
          isPro: false
          name: Kamaljp
          type: user
        html: '<p>Did you guys try to run it locally by pulling the model pipeline?
          Say in Google Colab Notebook..<br>The model is just 1.5GB, so it must easily
          load there... </p>

          <p>It takes four lines code...</p>

          <p>cell1: !pip install -q transformers datasets torch &gt; /dev/null</p>

          <p>cell2: from transformers import pipeline</p>

          <p>cell3: img2text = pipeline(task=''image-to-text'',model=''google/pix2struct-textcaps-base'')</p>

          <p>cell4: img2text("/content/your_image.png")</p>

          <p>It works, I checked it with a thumbnail of the below video<br><a rel="nofollow"
          href="https://youtu.be/tjrdb8tdXT4">https://youtu.be/tjrdb8tdXT4</a></p>

          '
        raw: "Did you guys try to run it locally by pulling the model pipeline? Say\
          \ in Google Colab Notebook..\nThe model is just 1.5GB, so it must easily\
          \ load there... \n\nIt takes four lines code...\n\ncell1: !pip install -q\
          \ transformers datasets torch > /dev/null\n\ncell2: from transformers import\
          \ pipeline\n\ncell3: img2text = pipeline(task='image-to-text',model='google/pix2struct-textcaps-base')\n\
          \ncell4: img2text(\"/content/your_image.png\")\n\nIt works, I checked it\
          \ with a thumbnail of the below video\nhttps://youtu.be/tjrdb8tdXT4"
        updatedAt: '2023-05-09T08:15:07.371Z'
      numEdits: 0
      reactions: []
    id: 645a010b2829ab9e92494ed9
    type: comment
  author: Kamaljp
  content: "Did you guys try to run it locally by pulling the model pipeline? Say\
    \ in Google Colab Notebook..\nThe model is just 1.5GB, so it must easily load\
    \ there... \n\nIt takes four lines code...\n\ncell1: !pip install -q transformers\
    \ datasets torch > /dev/null\n\ncell2: from transformers import pipeline\n\ncell3:\
    \ img2text = pipeline(task='image-to-text',model='google/pix2struct-textcaps-base')\n\
    \ncell4: img2text(\"/content/your_image.png\")\n\nIt works, I checked it with\
    \ a thumbnail of the below video\nhttps://youtu.be/tjrdb8tdXT4"
  created_at: 2023-05-09 07:15:07+00:00
  edited: false
  hidden: false
  id: 645a010b2829ab9e92494ed9
  type: comment
is_pull_request: false
merge_commit_oid: null
num: 2
repo_id: google/pix2struct-textcaps-base
repo_type: model
status: open
target_branch: null
title: The model_type 'pix2struct' is not recognized. It could be a bleeding edge
  model, or incorrect
